modules/serverless-service-mesh-resources.adoc

```diff
@@ -127,3 +129,8 @@ spec:
 ----
 $ oc apply -n istio-system -f <filename>
 ----
+
+[IMPORTANT]
+====
+{product-title} route must be created in the same namespace with your ServiceMeshControlPlane. In this example, ServiceMeshControlPlane is deployed in `istio-system` namespace.
```

Comment:
```suggestion
The {product-title} route must be created in the same namespace as the ServiceMeshControlPlane. In this example, the ServiceMeshControlPlane is deployed in the `istio-system` namespace.
```

Reply:
@nak3 lmk whenever this is fixed and I'll ping the docs folks for review and merge. Thanks!

Reply:
Thank you @abrennan89 .  Updated!